app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt

```diff
@@ -35,6 +37,13 @@ class HomeActivityTestRule(
         if (skipOnboarding) { skipOnboardingBeforeLaunch() }
     }
 
+    override fun afterActivityLaunched() {
+        super.afterActivityLaunched()
+        if (skipShouldShowJumpBackIn) {
+            activity.settings().shouldShowJumpBackInCFR = false
```

Comment:
question: why is this in `afterActivityLaunched` but the prior other invocation is in `beforeActivityLaunched`?

Reply:
Because the activity is null in `beforeActivityLaunched` and it needs to be instantiated to get a handle on `settings` 😄

Reply:
Nevermind, I reverted it and its working now . I'll leave it in the `beforeActivityLaunched`